sim/team-validator.ts

```diff
@@ -569,7 +573,7 @@ export class TeamValidator {
 		const lsetProblems = this.reconcileLearnset(outOfBattleSpecies, setSources, lsetProblem, name);
 		if (lsetProblems) problems.push(...lsetProblems);
 
-		if (ruleTable.has('obtainablemisc') && outOfBattleSpecies.forme?.includes('Gmax')) {
+		if (ruleTable.has('obtainablemisc') && set.gigantamax) {
 			if (!setSources.sourcesBefore) {
 				problems.push(`${name} has an exclusive move that it doesn't qualify for (because Gmax Pokemon can only be obtained from a Max Raid).`);
```

Comment:
Well this isn't true any more is it?

Reply:
Probably not, but I wanted to touch validation as little as possible

Reply:
I would add a flag for this so we could simulate the old behavior, for the record. But yeah, probably a separate PR.